src/components/Table/Table/__stories__/table.stories.mdx

```diff
@@ -349,6 +348,48 @@ This is an example of a table with 5000 rows
   </Story>
 </Canvas>
 
+## [Clickable-highlighted rows - demo story]
```

Comment:
**TODO remove demo story**

Reply:
the story should be replaced with a simple table with a few lines highlighting its first row.
no clicking